tests/test_orchestration/test_orchestrator.py

```
@@ -200,6 +201,52 @@ def test_empty_list(self):
 
+class TestStartProxyFinishReason:
+    def _run(self, svc: object) -> list[str]:
+        import nemo_evaluator.adapters.proxy as proxy_mod
+        from nemo_evaluator.orchestration.orchestrator import _start_proxy
+
+        captured: dict[str, Any] = {}
+
+        class _Handle:
+            url = "http://127.0.0.1:0"
+
+        def _fake_start(**kwargs: Any) -> _Handle:
```

Ruff (0.15.18)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)